packages/carbon-react/src/components/Pagination/Pagination.stories.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import { action } from '@storybook/addon-actions';
+
+import {
+  withKnobs,
+  array,
+  boolean,
+  number,
+  text,
+} from '@storybook/addon-knobs';
```

Comment:
Can we remove this? We'll go through and add the new v6 controls at a later date since the knobs don't seem to be working with the new storybook